include/swift/AST/Decl.h

```diff
@@ -2166,14 +2163,7 @@ class ValueDecl : public Decl {
   /// taken into account.
   ///
   /// \sa getFormalAccessScope
-  Accessibility getFormalAccess(const DeclContext *useDC = nullptr) const {
-    assert(hasAccessibility() && "accessibility not computed yet");
-    Accessibility result = TypeAndAccess.getInt().getValue();
-    if (useDC && (result == Accessibility::Internal ||
-                  result == Accessibility::Public))
-      return getFormalAccessImpl(useDC);
-    return result;
-  }
+  Accessibility getFormalAccess(const DeclContext *useDC = nullptr) const;
```

Comment:
This split was deliberate, so that an already-computed access could be inlined into the caller. It might not be worth it, though.